SourceX/controls/devices/joystick.cpp

```diff
@@ -4,7 +4,7 @@
 #include "controls/controller_motion.h"
 #include "stubs.h"
 
-#ifdef SWITCH
+#ifdef __SWITCH__

```

Comment:
We may want to move these to `CMakeLists.txt`

Reply:
@glebm would you be willing to make this change?

Reply:
https://github.com/diasurgical/devilutionX/compare/d1fbb5fe6e10d354d43326a813b95052b7e24476..846ecf36fb849c645d1c362d66c2c9b7031d2b82